controller/dash_circuit_board.py

```diff
@@ -0,0 +1,66 @@
+import configurations.youtube as yt
+import configurations.netflix as nf
+import configurations.amazon_prime as ap
+
+class DashCircuitBoard:
+    ''' Configure gestures to '''
+
+    def __init__(self):
+        self.config_name = 'init'
+        self.config_class = None
+        self.config_obj = None
+        self.context_pool = ['init']
```

Comment:
Context Pool? is it to keep track of current config &/or circuit for Youtube/Netflix/Amazon_Prime?